gcp-jobs/update-legal-filings/src/update_legal_filings/config.py

```diff
@@ -26,15 +26,21 @@ class _Config:  # pylint: disable=too-few-public-methods
 
     PROJECT_ROOT = os.path.abspath(os.path.dirname(__file__))
 
+    ENABLE_BN_UPDATES = bool(os.getenv("ENABLE_BN_UPDATES", "True"))
```

Comment:
```suggestion
    ENABLE_BN_UPDATES = os.getenv("ENABLE_BN_UPDATES", "True") == "True"
```
![image](https://github.com/user-attachments/assets/cf10ecc8-be57-4acd-a2fe-adc1e66baebc)

Reply:
Nice catch, thanks